### DISCUSSION TITLE
[UE]有没有DTS的生成黑名单之类的功能？

### DISCUSSION THREAD

**Discussion by enixfu** [Tencent/puerts#888] [Q&A] [answered]
美术提交了很多测试蓝图，全是编译错误，有没有黑名单之类的功能可以跳过这些错误蓝图不生成DTS？

**Comment by mysticfarer** [comment, marked as answer]
新版本已经有了，已经在使用：

https://github.com/Tencent/puerts/blob/e702975fe7362c5359e5d3d6d1176139918509f0/unreal/Puerts/Source/Puerts/Private/PuertsSetting.h#L48
